content/en/cloud_cost_management/tag_pipelines.md

```diff
@@ -56,14 +61,19 @@ To ensure the rule only applies if the `application` tag doesn't already exist,
 
 ### Map multiple tags
 
-Use [Reference Tables][2] to add multiple tags to cost data without creating multiple rules. This will map the values from your Reference Table's primary key column to values from cost tags. If found, the pipelines adds the selected Reference Table columns as tags to cost data.
+Use [Reference Tables][2] to add multiple tags to cost data without creating multiple rules. This maps the values from your Reference Table's primary key column to values from cost tags. If found, the pipelines adds the selected Reference Table columns as tags to cost data.
 
 For example, if you want to add information about which VPs, organizations, and business_units different AWS and Azure accounts fall under, you can create a table and map the tags.
 
-{{< img src="cloud_cost/tags_mapmultiple.png" alt="Add account metadata like vp, organization, and businessunit using reference tables for tag pipelines" style="width:60%;" >}}
+{{< img src="cloud_cost/tag-pipelines-multiple-tags.png" alt="Add account metadata like customer_name using reference tables for tag pipelines" style="width:60%;" >}}
 
 Similar to [Alias tag keys](#alias-tag-keys), the rule stops executing for each resource after a first match is found. For example, if an `aws_member_account_id` is found, then the rule no longer attempts to find a `subscriptionid`.
 
+Under the **Additional options** section, you have the following options:
+
+- **Only apply if columns don't exist** - Ensures the defined columns are only added if they do not already exist with the associated tags from the cost data.
+- **Apply case-insensitive matching for primary key values** - Enables the primary key value from the reference table and tag value from the primary key tag from the cost data to be case insensitive. For example: if the primary key value pair from UI is: foo:Bar, the tag from cost data is foo:bar, then the two can be matched.
```

Comment:
```suggestion
- **Apply case-insensitive matching for primary key values** - Enables case-insensitive matching between the primary key value from the reference table and the value of the tag in the cost data where the tag key matches the primary key. For example, if the primary key value pair from UI is: foo:Bar and the tag from the cost data is foo:bar, then the two can be matched.
```

Reply:
I'm not sure my suggestion improves things, but I shared it because I found myself rereading the original sentence a few times.